### PR TITLE
Use zimscraperlib.video to transcode videos to WebM

### LINKED ISSUE #33: Use video convertion from zimscraperlib
We have option to convert the video (downloaded in a video xblock) into WebM. We shall rather use zimscraperlib.video for the task.

### PR DESCRIPTION
**Pull request by satyamtg**
This fixes #33 by using zimscraperlib.video module to convert to WebM

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
         "mistune",
         "requests",
         "iso-639",
+        "zimscraperlib",
```
Comment (rgaudin) [inline setup.py:28]: Can we have a versioned line?

**Comment by rgaudin** [comment]
Good. Just version the zimscraperlib so it doesn't explode in the future. We could do same for other requirements as well.

**Comment by satyamtg** [comment]
> Good. Just version the zimscraperlib so it doesn't explode in the future. We could do same for other requirements as well.

https://github.com/openzim/openedx/tree/refactored_project_structure - This branch actually uses a requirements.txt file with correct versions for everything. I thought we can close #37 along with #40 with that branch.

**Comment by rgaudin** [comment]
I don't see how that prevents us from adding the version here. That other branch doesn't even have a PR

File: setup.py [hunk 2]
```diff
@@ -15,17 +15,18 @@
...
+        "lxml==4.5.1",
+        "webvtt-py==0.4.5",
+        "python-slugify==4.0.0",
+        "Jinja2==2.11.2",
+        "mistune==0.8.4",
+        "requests>=2.24,<3.0",
+        "iso-639==0.4.5",
+        "zimscraperlib>=1.1.2,<1.2",
+        "python-magic==0.4.18",
+        "youtube_dl",
```
Comment (rgaudin) [inline setup.py:29]: Do we still need youtube_dl then?

File: setup.py (hunk 1 above)
Comment (satyamtg) [reply setup.py]: Done

File: setup.py (hunk 2 above)
Comment (satyamtg) [reply setup.py:28]: Yup. We use it to download some videos in some courses as they are hosted on youtube

Comment (satyamtg) [reply setup.py:28]: See https://github.com/openzim/openedx/blob/155a79b3b134649209db023ebe2d20f6d0e8a1a7/openedxtozim/xblocks_extractor/Video.py#L108

and 

https://github.com/openzim/openedx/blob/155a79b3b134649209db023ebe2d20f6d0e8a1a7/openedxtozim/utils.py#L215

**Pull request merged**